readme.md

```diff
@@ -9,6 +9,7 @@ Multi-platform and multi-language.
 [![codecov](https://codecov.io/gh/renovatebot/renovate/branch/main/graph/badge.svg)](https://codecov.io/gh/renovatebot/renovate)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![Build status](https://github.com/renovatebot/renovate/workflows/build/badge.svg)](https://github.com/renovatebot/renovate/actions)
+![Docker Pulls](https://img.shields.io/docker/pulls/renovate/renovate?color=turquoise)
```

Comment:
why your using a different color?

Reply:
This badge doesn't have a positive or negative outcome / range like the codecov or build badges so I didn't go with the red or green colors. I chose the turquoise color since it's the closest I found to the Docker colors, but it's really not a critical detail if you feel it should be a different color.